### PR TITLE
Implemented support to store user preference for selected product store (#2f2h8hu)

### PR DESCRIPTION
**Pull request by rathoreprashant**
### Related Issues

Closes #

### Short Description and Why It's Useful

Added user preference for selected product store

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)
- [ ] I have changed the target to hacktoberfest branch

### CODE REVIEW

File: changelogs/unreleased/-251xhvt.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Implemented functionality to add support to select eComStore from the settings page.
+ticket_id: "#251xhvt"
```
Comment (ymaheshwari1) [inline changelogs/unreleased/-251xhvt.yml:3]: Remove changelog file

File: src/services/UserService.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import api from '@/api'
+import { client } from '@/api';
```
Comment (ymaheshwari1) [inline src/services/UserService.ts:2]: We can all the imports from `@/api` in single line.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: changelogs/unreleased/-251xhvt.yml (hunk 1 above)
Comment (rathoreprashant) [reply changelogs/unreleased/-251xhvt.yml]: Removed sir

File: src/services/UserService.ts (hunk 2 above)
Comment (rathoreprashant) [reply src/services/UserService.ts]: Updated

**Comment by dt2patel** [comment]
not using product store in inventory count app. closing PR

**Pull request closed without merging**